examples/centralWidget/digitalclock.h

```diff
@@ -0,0 +1,17 @@
+#ifndef DIGITALCLOCK_H
+#define DIGITALCLOCK_H
+
+#include <QLCDNumber>
+
+class CDigitalClock : public QLCDNumber
```

Comment:
Please remove all fixed height widgets from the example (the top one and the bottom status bar) and remove the digitalclock source files.